e2e/pull/wksp/MODULE.bazel

```diff
@@ -11,7 +11,7 @@ local_path_override(
 oci = use_extension("@rules_oci//oci:extensions.bzl", "oci")
 oci.pull(
     name = "empty_image",
-    digest = "sha256:2d4595bbc0fabeb1489b1071f56c26f44a2f495afaa9386ad7d24e7b3d8dfd3e",
+    digest = "sha256:f7f4df35452e2f7fa0906fd45902f46b35e0c6e95d41ed891df31f4937ba1632",
```

Comment:
are you forced to update these?

Reply:
Yes, because digests created in 1.x is different than 2.x, these should be stable from now, if not it'll catch some accidental changes we might make that changes the digest  and not realize.